station/app/schemas/local_trains.py

```diff
@@ -27,25 +27,87 @@ class LocalTrain(DBSchema):
 
 
 class LocalTrainCreate(BaseModel):
-    train_name: str
+    train_name: Optional[str] = None
 
+class LocalTrainConfigBase(BaseModel):
+    name: str
+    image: Optional[str] = None
+    tag: Optional[str] = None
+    query: Optional[str] = None
+    entrypoint: Optional[str] = None
+    volumes: Optional[str] = None
+    env: Optional[str] = None
+    train_id: Optional[str] = None
```

Comment:
i think things like image, tag,  query, entrypoint are properties of the and should probably be part of the train not the config